fetch.bs

```diff
@@ -1973,6 +1973,10 @@ this is also tracked internally using the request's <a for=request>timing allow
 <dfn for=response id=concept-response-timing-info>timing info</dfn> (null or a
 <a for=/>fetch timing info</a>), which is initially null.
 
+<p>A <a for=/>response</a> has an associated
+<dfn export for=response id=concept-response-service-worker-timing-info>service worker timing</dfn> (null or a
```

Comment:
Exceeds 100 columns.

Reply:
Also no need for the ID here and I wonder if we should name this field "service worker timing info" for consistency with "timing info".